docs/source/llms/tracing/tracing-schema.rst

```diff
@@ -0,0 +1,328 @@
+              </p>
+            </div>
+            <div class="image-box">
+                <img src="../../_static/images/llms/tracing/schema/trace_data_architecture.png"/>
+            </div>
+        </div>
+
+.. |span-architecture| raw:: html
+
+        <div class=""main-container"">
+            <div>
+              <h4>Span Structure</h4>
+              <p> The Span object within MLflow's tracing feature provides detailed information about the individual steps of the trace. 
+                Each Span object contains information about the step being instrumented, including the span_id, name, start_time, parent_id, status, 
+                inputs, outputs, attributes, and events.
+              </p>
+            </div>
+            <div class="image-box">
+                <img src="../../_static/images/llms/tracing/schema/span_architecture.png"/>
+            </div>
+        </div>
+
+
+.. container:: tracing-responsive-tabs
+
+    .. tabs::
+
+        .. tab:: Trace Architecture
+
+            |trace-architecture|
+
+        .. tab:: Trace Info
+
+            |trace-info|
+
+        .. tab:: Trace Data
+
+            |trace-data|
+
+        .. tab:: Span Architecture
+
+            |span-architecture|
+
+
+Trace Schema
+------------
+
+A trace is composed of two components:
+
+- :py:func:`mlflow.entities.trace_info.TraceInfo`
+
+- :py:func:`mlflow.entities.trace_data.TraceData`
+
+.. tip::
+    Check the API documentation for helper methods on these dataclass objects for more information on how to convert or extract data from them.
+
+
+Trace Info
+----------
+
+Trace Info is a dataclass object that contains metadata about the trace. This metadata includes information about the trace's origin, status, and 
+various other data that aids in retrieving and filtering traces when used with :py:meth:`mlflow.client.MlflowClient.search_traces` and for 
+navigation of traces within the MLflow UI.
+
+To learn more about how ``TraceInfo`` metadata is used for searching, you can see examples :ref:`here <search_traces>`.
+
+The data that is contained in the ``TraceInfo`` object is used to populate the trace view page within the MLflow tracking UI, as shown below.
+
+.. figure:: ../../_static/images/llms/tracing/schema/trace_info_in_ui.png
+    :alt: TraceInfo as it is used in the MLflow UI
+    :width: 100%
+    :align: center
+
+The primary components of MLflow :py:class:`mlflow.entities.trace_info.TraceInfo` objects are listed below.
+
+Request ID
+^^^^^^^^^^
+
+The ``request_id`` of a trace is a unique identifier that is generated for each trace. This identifier is used within MLflow and integrated systems to 
+resolve the event being captured and to provide associations for external systems to map the logged trace to the originating caller. 
+
+.. note::
+    In the fluent API, this value will be generated for you. In the client API, you can provide a request_id to associate with the trace or omit 
```

Comment:
This is a bit inaccurate. Regardless of which API they use, the request ID must be generated by MLflow (`MlflowClieit().start_trace()` generates request ID). What users can control is which generated ID they can pass to each spans.

Reply:
ahhh yeah. Fixed this